pkg/db/canary.go

```diff
@@ -48,7 +48,8 @@ func GetAllCanaries() ([]v1.Canary, error) {
 	for _, _canary := range _canaries {
 		c, err := _canary.ToV1()
 		if err != nil {
-			return nil, err
+			logger.Errorf("Error parsing canary[%s]: %v", _canary.ID, err)
```

Comment:
@yashmehrotra  Can we add a job history event as well ?

Reply:
Have added